hermes/probe/read/read_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Alicja Kwiecinska, GitHub: alicjakwie
+
+package read
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/create"
+	"github.com/googleinterns/step224-2020/hermes/probe/fakegcs"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+
+	metricpb "github.com/google/cloudprober/metrics/proto"
+	probepb "github.com/googleinterns/step224-2020/config/proto"
+	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+const (
+	fileSizeBytes = 100
+)
+
+func TestReadFile(t *testing.T) {
+	target := &probe.Target{
+		&probepb.Target{
+			Name:                   "hermes",
+			TargetSystem:           probepb.Target_GOOGLE_CLOUD_STORAGE,
+			TotalSpaceAllocatedMib: int64(1000),
```

Comment:
In here to you mean Mib (Mebibit) or MiB (Mebibyte)? Usually storage information is represented in bytes not bits. If you meant bytes change this to `TotalSpaceAllocatedMiB`.

Since it appears that this is in another file, you can just open an issue and add a to-do.

Reply:
added a TODO: #79